DMX.Sdk.Tests.Unit/Services/Foundations/Labs/LabServiceTests.Logic.RetrieveAll.cs

```diff
@@ -0,0 +1,41 @@
+﻿// --------------------------------------------------------------- 
+// Copyright (c) Microsoft Corporation. All rights reserved. 
+// ---------------------------------------------------------------
+
+using DMX.Sdk.Models.Services.Foundations.Labs;
+using FluentAssertions;
+using Force.DeepCloner;
+using Moq;
+using Xunit;
+
+namespace DMX.Sdk.Tests.Unit.Services.Foundations.Labs
+{
+    public partial class LabServiceTests
+    {
+        [Fact]
+        public async Task ShouldRetrieveAllLabsAsync()
+        {
+            // given
+            List<Lab> randomLabs = CreateRandomLabs();
+            var returnedLabs = randomLabs;

```

Comment:
If the right side isn't clear as in `new Something();` you must specify the type and not use `var`

Read more here:
https://github.com/hassanhabib/CSharpCodingStandard/blob/master/5.%20Variables.md#011-semi-clear-types